src/chat/narrowsSelectors.js

```diff
@@ -104,47 +104,46 @@ export const getRecipientsInGroupNarrow: Selector<UserOrBot[], Narrow> = createS
 );
 
 // TODO: clean up what this returns.
-export const getStreamInNarrow = (
-  narrow: Narrow,
-): Selector<Subscription | {| ...Stream, in_home_view: boolean |}> =>
-  createSelector(
-    getSubscriptions,
-    getStreams,
-    (subscriptions, streams) => {
-      if (!isStreamOrTopicNarrow(narrow)) {
-        return NULL_SUBSCRIPTION;
-      }
+// prettier-ignore
```

Comment:
What's the reason for the prettier-ignore?

These are definitely sometimes the right thing -- just curious what the reason is in this instance.

Reply:
as below line is too long, prettier splits it into two line like:
```
export const getStreamInNarrow: Selector<Subscription | {| ...Stream, in_home_view: boolean |},
  Narrow, > = createSelector(
  (state, narrow) => narrow,
```

And then lint complains:
```
Running lint

  107:41  error  There must be no space at end of "Selector" generic type annotation  flowtype/generic-spacing
```

Reply:
Got it. The same issue as in a couple of other spots in this file, then:
```js
// Prettier mishandles this Flow syntax.
// prettier-ignore
export const getShownMessagesForNarrow: Selector<$ReadOnlyArray<Message | Outbox>, Narrow> =
```

I'll add the same comment here.